bentoml/_internal/runner/runner.py

```diff
@@ -63,6 +63,10 @@ class Runner:
 
     _runner_handle: RunnerHandle = attr.field(init=False, factory=DummyRunnerHandle)
 
+    if TYPE_CHECKING:
+        run: t.Callable[..., t.Any]
+        async_run: t.Callable[..., t.Awaitable[t.Any]]
```

Comment:
This change looks fine to me but adding the return type as `t.Any` isn't the most useful. Is it possible to derive the return types based on the framework runner implementation?

Reply:
I think `t.Any` is the best effort here. Another way to do this is to infer the return type from `@svc.api` decorated function.

Reply:
for the suggestion we can do
```
import inspect
inspect.get_annotations(pipeline.runnable_class.bentoml_runnable_methods__["__call__"].func)
```

Reply:
We will most likely invest more into runner interface declaration when we design the different runtime integration. I'd recommend not to invest too heavily into deriving the runner signature. The best effort here is good for now.